pkg/utils/condition/condition.go

```diff
@@ -342,28 +342,28 @@ func (c ResourceCondition) TrueClusterResourcePlacementCondition(generation int6
 			Status:             metav1.ConditionTrue,
 			Type:               string(fleetv1beta1.ClusterResourcePlacementOverriddenConditionType),
 			Reason:             OverriddenSucceededReason,
-			Message:            fmt.Sprintf("The selected resources are successfully overridden in the %d clusters", clusterCount),
+			Message:            fmt.Sprintf("The selected resources are successfully overridden in %d cluster(s)", clusterCount),
 			ObservedGeneration: generation,
 		},
 		{
 			Status:             metav1.ConditionTrue,
 			Type:               string(fleetv1beta1.ClusterResourcePlacementWorkSynchronizedConditionType),
 			Reason:             WorkSynchronizedReason,
-			Message:            fmt.Sprintf("Works(s) are succcesfully created or updated in the %d target clusters' namespaces", clusterCount),
+			Message:            fmt.Sprintf("Works(s) are succcesfully created or updated in %d target clusters' namespaces", clusterCount),
```

Comment:
```suggestion
			Message:            fmt.Sprintf("Works(s) are succcesfully created or updated in %d target cluster(s)' namespaces", clusterCount),
```

Reply:
Should not be on this PR, fix will be reflected in #806.